README.md

```diff
@@ -1,2 +1,33 @@
 # flask-container-scaffold
 Common base layer for Flask applications that are deployed in containers.
+
+
+### Setting up a development environment
+
+You may set up your environment with virtualenv or another preferred tool for
+managing virtual environments, but here are some directions for doing so using
+[pipenv](https://pipenv.pypa.io/en/latest/). First, install pipenv:
+  pip install --user pipenv
+
+Next, using it to set p your development environment:
+  pipenv update -d
+
+If you prefer to use pip directly in your venv, simply specify the
+requirements.txt and test-requirements.txt files.  There is also a
+dist-requirements.txt, if you will be building the project for distribution.
+
+Any remaing directions will assume you are either in your venv, which for
+pipenv, can be activated like this:
+  pipenv shell
+Alternatively, any commands can be run in you pipenv venv by prepending with:
+  pipenv run
```

Comment:
It's pretty minor but it would be nice to correct the markdown spacing/syntax around the commands in this document so they are formatted like actual commands and easier to read. It was still mostly readable for the other ones, but because there's no punctuation around this one [it looks a bit more messy and less readable](https://github.com/jguiditta/flask-container-scaffold/tree/extract_app_scaffold#setting-up-a-development-environment), in my opinion.

Also, "in you pipenv" -> "in your pipenv"